### ISSUE TITLE
Sprint 3 Retrospective

### ISSUE THREAD

**Issue by NoahMcKearney** [aaliyahortiz26/snhuSearch#43]
1. What went well during this last sprint?
2. What did not go well during this last sprint?
3. What can we do better next sprint to improve our process?

**Comment by aaliyahortiz26** [comment]
For question 1: Things that went well during this sprint is that we were much better at getting tasks done more on-time up until about a little over the half way mark which is where we slowed down. However, even after that we became better at uploading smaller bits of work to help others get around their blockers somewhat and as a team we adjusted to blockers way better than previous sprints this time around.

For question 2: Things that did not go well during this past sprint we would say was definitely communication in the later parts of the sprint which caused some disorganisation amongst peers. Additionally we struggled with metric beat, and a lack of support on connecting C# to python in an efficient manner.

For question 3: Things we could definitely improve upon following up is still communication. Despite us working on our tasks this is something that we could definitely do better on as a group and it would greatly benefit everyone in general. Besides communication, we could also still be splitting up tasks more evenly and dividing them more evenly to better disburse our work and time.
